src/executor/infer_graph_attr_pass.cc

```diff
@@ -254,7 +254,8 @@ nnvm::Graph InferAttr(nnvm::Graph &&ret,
         dispatch_mode = &dispatch_modes[nid];
         if (dispatch_modes[nid] == DispatchMode::kUndefined) forward_known = false;
       }
-      auto finfer = finfer_shape.get(inode.source->op(), fdefault);
+      auto finfer = (inode.source->op() == Op::Get("_zeros")) ? fdefault :
+        finfer_shape.get(inode.source->op(), fdefault);
```

Comment:
Are you sure about this? This affects all _zero ops, not just for the case you mentioned.

Reply:
You are right, this is breaking some unit test (however, due to unittest of master branch is broken in MacOS, I wan't able to verify before checkin). I have changed the PR to WIP.